db/schema.rb

```diff
@@ -51,7 +51,6 @@
     t.string "prisoner_category"
     t.string "sentence_length"
     t.jsonb "complex_cases"
-    t.integer "moves_count", null: false
```

Comment:
we shouldn't commit those, just did a drop/create/migrate and those shouldn't move around

Reply:
Good spot - reverted 👍